test/arithmetic/floating_point/floating_point_sqrt_test.dart

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Indentifier: BSD-3-Clause
+//
+// floating_point_sqrt.dart
+// An abstract base class defining the API for floating-point square root.
+//
+// 2025 March 5
+// Authors: James Farwell <james.c.farwell@intel.com>,
+//Stephen Weeks <stephen.weeks@intel.com>,
+//Curtis Anderson <curtis.anders@intel.com>
+
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/src/arithmetic/arithmetic.dart';
+import 'package:test/test.dart';
+
+void main() {
+  tearDown(() async {
+    await Simulator.reset();
+  });
+  test('FP: square root with non-FP numbers', () {
+    // building with 16-bit FP representation
+    const exponentWidth = 3;
+    const mantissaWidth = 7;
+
+    final fv = FloatingPointValue(
```

Comment:
This is an odd value leading the reader to believe it is important.  Why not just populate(..).zero;